pkgs/tools/video/gopro/default.nix

```
@@ -0,0 +1,35 @@
+{ stdenv, fetchFromGitHub
+, ffmpeg
+, imagemagick
+, makeWrapper
+, mplayer
+}:
+
+stdenv.mkDerivation rec {
+  pname = "gopro";
+  version = "1.0";
+
+  src = fetchFromGitHub {
+    owner = "KonradIT";
+    repo = "gopro-linux";
+    rev = version;
+    sha256 = "0sb9vpiadrq8g4ag828h8mvq01fg0306j0wjwkxdmwfqync1128l";
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    chmod +x gopro
+    cp gopro $out/bin
+    wrapProgram $out/bin/gopro \
+      --suffix-each PATH ":" "${ffmpeg}/bin ${imagemagick}/bin ${mplayer}/bin"
```

Usually we use `--prefix` in nixpkgs. Is there a reason why you went with `--suffix-each`?
```suggestion
      --prefix PATH : "${stdenv.lib.makeBinPath [ ffmpeg imagemagick mplayer ]}"
```

---

nothing in particular, was looking at what I could give to wrapProgram, and it seemed to fit my use case.